tools/third_party.py

```diff
@@ -110,6 +139,27 @@ def delete_lockfile():
     delete_lockfile()
 
 
+# Install python packages with pip.
+def run_pip():
+    # Upgrade pip itself. The version that is bundled with python is too old
+    # to support the next steps.
+    run([
+        sys.executable, "-m", "pip", "install", "--upgrade", "--target",
+        python_packages_path, "pip"
+    ],
+        cwd=third_party_path,
+        env=python_env())
+
+    # Install pywin32.
+    run([
+        sys.executable, "-m", "pip", "install", "--upgrade", "--target",
+        python_packages_path, "--platform=win_amd64", "--only-binary=:all:",
+        "pypiwin32"
+    ],
```

Comment:
What is the purpose of "pypiwin32" ? And why does it need to be "pip installed" if its being bundled?